### DISCUSSION TITLE
Access to Copilot

### DISCUSSION THREAD

**Discussion by Behachee** [community/community#109552] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I cannot use Copilot anymore since today. I have this error message, but when I go to my github profil it says that I have Copilot activated for this account.

Error Message:

2024-02-22 12:12:58.238 [info] [auth] Invalid copilot token: missing token: 403 
2024-02-22 12:12:58.252 [error] [default] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as TheoBh."

**Comment by gil-son** [comment]
Hello,

TheoBh... is your current account on Github?

**Reply by Behachee** [reply]
Hello, 

My account linked with Copilot is Behachee, the one I'm currently writting with. But don't know how to link Copitlot with this account.

**Comment by davecheney** [comment, marked as answer]
@Behachee 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by Behachee** [reply]
Thanks it works ! :)
